roles/swift-account/tasks/main.yml

```diff
@@ -1,5 +1,5 @@
 ---
-- name: swift-account service scripts
+- name: swift-account service scripts (ubuntu)
```

Comment:
can we follow other roles, where service parameters are defined in openstack-meta and used here.